src/praisonai/praisonai/tools/kanban_tools.py

```
@@ -163,42 +173,98 @@ def kanban_show(task_id: str) -> Dict[str, Any]:
         store = _get_kanban_store()
         
+        # Record a completed run capturing the structured handoff
+        run = None
+        try:
+            run = store.record_run(
+                task_id,
+                'completed',
+                summary=summary,
+                metadata=metadata or {},
+            )
+        except Exception as run_err:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)